esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -12,6 +12,7 @@
 from dask import array as da
 
 from esmvalcore._config import get_tag_value
+from esmvalcore._citation import cite_tag_value
```

Comment:
It would be better to just define the `cite_tag_value` function in this file, because this is the only place where it is needed.